packages/core/src/Stores/client-store.js

```diff
@@ -725,6 +726,11 @@ export default class ClientStore extends BaseStore {
         return typeof client_tnc_status !== 'undefined' && client_tnc_status !== terms_conditions_version;
     }
 
+    get is_tnc_status_updated() {
```

Comment:
can we create a hook for checking `is_tnc_status_updated` instead of adding it in client store?

Reply:
I will check on this 🙏